### ISSUE TITLE
Campus contacts

### ISSUE THREAD

**Issue by stephlabou** [uc-love-data-week/uc-love-data-week.github.io#132]
Feedback from UC Love Data Week planning group:

- I don't really love the black background with light gray text for accessibility reasons. Do you think it is possible to make this section look more like the participating campuses section?
- Also, it would be helpful to add hyperlinks to each campus contact email address.

**Comment by kekoziar** [comment]
> @kekoziar my memory is that we don't do hyperlinks for emails or otherwise make them clickable, to reduce spam emails / bots. Is that right?

@stephlabou yes. it's also why the email addresses use the written word `<at>` instead of the symbol.

**Comment by stephlabou** [comment]
Thanks for confirming and it'll be good to have this documented in this issue in case the question comes up again. I'll leave the issue open until we resolve the question of background color of campus contents section.

**Comment by stephlabou** [comment]
Addressed in [PR #134](https://github.com/uc-love-data-week/uc-love-data-week.github.io/pull/134). Removing global map image should help with readability for campus contacts. Note that this changes footer for all prior years as well, since they are all built off 2016 CSS files.
